### PR TITLE
Contact-form 

### PR DESCRIPTION
**Pull request by Frelly0**
Accomplishment
It is my personal portfolio project and i have added new feature in repository which are:
 . added contact html
 . added  contact css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,190 +1,240 @@
...
\ No newline at end of file
+    
+    <section class="contact">
+      <h1>Contact me</h1>
+      <p>
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it!
+      </p>
+      <form action="">
```
Comment (LionRouge1) [inline index.html:210]: - [ ] Nice work implementing the form, it look really good :clap: However it states in the project requirement that data should be collected by using the [Formspree service](https://formspree.io/html). Therefore I suggest you:
  - add the `action` attribute to your form tag and its value should be similar to `https://formspree.io/f/{form_id}` where `{form_id}`  is `id` giving you when create an account on [formspree service](https://formspree.io/html).
  - add the `method` attribute to your `form` tag and its value  equals `POST`

File: index.html [hunk 2]
```diff
@@ -1,190 +1,240 @@
...
+        <input
+          type="text"
+          name="name"
+          id="name"
+          max="30"
```
Comment (LionRouge1) [inline index.html:215]: - [ ] Your name input it's take more than 30 chars. Kindly make sure it takes up to 30 chars as requested in the project requirement. You can fix that by changing the `max="30"` to `maxlength="30"`

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @Frelly0 & @mulugithub, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job implement the "contact me" section, it looks almost identical to the Figma design :1st_place_medal: 
- Great job using **flexbox** to lay out the "contact me" elements :+1: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -199,20 +199,20 @@ <h3>Skills</h3>
...
+
     <section class="contact">
       <h1>Contact me</h1>
       <p>
         If you have an application you are interested in developing, a feature
         that you need built or a project that needs coding. I’d love to help
         with it!
       </p>
-      <form action="">
+      <form action="https://formspree.io/f/{form_id}" method="post">
```
Comment (topeogunleye) [inline index.html:210]: * Great job here 💯, but you should Create a free account on [Formspree](https://formspree.io/) and choose New Form then Update your form's action. 
* [ ]  You have to replace "{form_id}" with the unique "form_id" which you will get after creating the account and form on [Formspree](https://formspree.io/) 👌
- [ ] Please make sure that the form is working as intended. You need to generate a Formspree endpoint and replace it with the `{form_id}`.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @Frelly0, @mulugithub,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* Changed spelling `max` to maxLength as suggested ✔️ 
* Good commit messages. :heavy_check_mark:
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- For future pull requests, please notice that your branch name `contact-css` does not follow the convention of being named after a feature or action. For example, you could name your branch `added-contact-css`. This makes it easier to track the changes and the purpose of each branch.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by osamaashraf6 — APPROVED** [review]
Hi @Frelly0  😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
MinionsGIF (2)

## Highlights🎉🎉

- [ ] All input fields are wrapped in a single-form element🚀
- [ ] All input fields have the correct associated placeholder text.💯
- [ ] Message input has maxlength="500" attribute.🚀
- [ ] All input fields have the required attribute to activate the validation on submission.👌


## Optional suggestions

- [ ] Please 😊 consider using best practices when you write commit messages, they should be capitalized sentences
- [ ] You can also consider 😃:

- For future pull requests, please notice that your branch name contact-css does not follow the convention of being named after a feature or action. For example, you could name your branch added-contact-css. This makes it easier to track the changes and the purpose of each branch.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**